action.yml

```
@@ -137,6 +137,10 @@ inputs:
     description: 'drift-detection slack drift url'
     required: false
     default: ''
+  cache-dependencies:
+    description: 'Leverage actions/cache to cache dependencies to speed up execution"
```

just got to testing this: you have a mismatched quotation mark here:

```suggestion
    description: "Leverage actions/cache to cache dependencies to speed up execution"
```